core/template.yaml

```diff
@@ -301,6 +301,43 @@ Resources:
       AliasName: !Sub alias/${AWS::StackName}/${Environment}/queue-kms-key
       TargetKeyId: !Ref SqsKmsKey
 
+  CslsKmsKey:
```

Comment:
is this just a generic key to encrypt s3 buckets? If so, then we could we make this a generic key? ie. call it `S3KmsKey`?